pyanaconda/core/configuration/payload.py

```diff
@@ -71,6 +71,21 @@ def enable_closest_mirror(self):
         """
         return self._get_option("enable_closest_mirror", bool)
 
+    @property
+    def enable_ignore_broken_packages(self):
+        """Enable --ignorebroken parameter of packages section.
+
+        This parameter could be dangerous for users because it will skip packages
+        with conflicting files so installed system may not contain all the required
+        functionality.
+
+        In case this feature will be disabled we won't start the installation when
+        --ignorebroken paramater is used.
```

Comment:
Again, this documentation should be more general and more focus on what happens when the configuration option is set to False or True. At the end, it is still the user who decides whether he wants to ignore the broken packages or not. It is not very clear from the description.

Reply:
Reworded.